store/streaming/plugins/abci/v1/README.md

```diff
@@ -0,0 +1,396 @@
+# ABCI and State Streaming Plugin (gRPC)
```

Comment:
what is the reasoning to have the import path be plugins/abci/v1? can we do only streaming since we expect the version in the sdk to only be plugin based

Reply:
The plugin system uses a handshake to make sure the correct plugin version is loaded. So it's important we version the ABCI listener service for future updates. I am also keeping in line with SDKs proto versioning scheme (although v1 may not be exactly correct).

Versioning may pan out not to be necessary in this case and we can remove it down the road but I don't see the harm in at least starting down this path. Ultimately, this is an SDK decision. I'll change it to what you decide but at a minimum we need to keep `streaming/abci` to distinguish between different plugins. 

plugin handshakes.
 https://github.com/cosmos/cosmos-sdk/blob/a36f16565381a2cf7deb7237e48ddaef8d96b7e6/store/streaming/streaming.go#L18-L20

Reply:
what other sorts plugins do you envision? semantic import could cause confusion with golang semantic confusion.

Reply:
Say that in the future you wanted to track some information regarding smart contracts. That could be a plugin `streaming/wasmd` etc. As to the semantic versioning I have no problem removing it and reducing to `streaming/abci`. Reducing it to `streaming/` would cause confusion in its own because the plugin system is built to support different plugins (protocols).

Reply:
Ready to fold this to `streaming/abci`?

Reply:
yea that works, thanks

Reply:
completed.